modules/auth/views/login.php

```diff
@@ -46,7 +46,7 @@
 
 
 	<fieldset type="buttons">
-		<input type="submit" value="Login">
+		<input type="submit" value="Log in">
 	</fieldset>
```

Comment:
Didn't we try this earlier..? I think the tests failed because it doesn't say "Login" anymore. We had trouble then to work with the test code, but see if you can find that.

Reply:
Yes, you are right. I have changed the test so I hope it will work.